compiler/rustc_hir_typeck/messages.ftl

```diff
@@ -18,7 +18,7 @@ hir_typeck_struct_expr_non_exhaustive =
     cannot create non-exhaustive {$what} using struct expression
 
 hir_typeck_method_call_on_unknown_type =
```

Comment:
I wonder if we should also rename this slug.. `method_call_on_unknown_raw_pointee`?